docs/tutorial/mlflow.rst

```diff
@@ -46,7 +46,7 @@ Enable the following MicroK8s addons to configure your Kubernetes cluster with e
 
 .. code-block:: bash
 
-   microk8s enable dns hostpath-storage ingress metallb:10.64.140.43-10.64.140.49
+   sudo microk8s enable dns hostpath-storage metallb:10.64.140.43-10.64.140.49 rbac
```

Comment:
If you installed `--classic` version, you do not need to use sudo here. Or not?
It's also recommended to enable plugins one by one, option to enable them all at once is deprecated and will be removed. Can we use:
```bash
microk8s enable dns
microk8s enable hostpath-storage
...
```